README.md

```diff
@@ -183,4 +208,12 @@ switch:
     switches:
       - port: 17
         name: "Speaker Relay"
-```
+
+
+# Add Debug info and issue reporting
+*Before* reporting issues please add this to your `configuration.yaml` `logger` section, check logs and report issue adding logging.
+```yaml
+logger:
+  default: info
+  logs:
+    custom_components.rpi_gpio: debug
```

Comment:
Please link to https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging instead. There is no need to use YAML anymore for debug and the UI functionality automatically download the log upon disable.

Reply:
done